src/site/artifacts/mod.rs

```diff
@@ -15,14 +17,25 @@ use serde::Serialize;
 type DownloadableFiles = Vec<(FileIdx, File, Vec<String>)>;
 /// A map from TargetTriples to Installers that support that platform
 type Platforms = BTreeMap<TargetTriple, Vec<InstallerIdx>>;
-#[derive(Serialize, Debug)]
+
+/// Current version of the JSON artifacts format
+static JSON_VERSION: usize = 1;
```

Comment:
is it expected for a user to have to deal with mixed versions of this? i.e. will they ever have archived json blobs from previous releases, or will they all get rebuilt together?